modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -20,18 +20,18 @@ def fetch_facilities_from_appointments(appointments)
 
           return nil unless facility_ids.any?
 
+          facility_ids = facility_ids.map do |facility|
+            Mobile::V0::Appointment.convert_from_non_prod_id!(facility)
+          end
+
           fetch_facilities_from_ids(facility_ids, nil)
         end
 
         def fetch_facilities_from_ids(facility_ids, include_children)
```

Comment:
So this method is also being called from the facilities info controller. Just to be clear, we do not want this conversion happening in that controller? That's what it looks like from your specs, so I'm just confirming.

Reply:
the method `fetch_facilities_from_ids` is being called both for the appointment index and facilities info index but we don't want the conversion for facilities info, we want to keep the 9xx ids. so this moves that conversion to only happen for appointment index